dev/com.ibm.ws.jsfContainer/src/com/ibm/ws/jsf/container/cdi/IBMViewHandlerProxy.java

```diff
@@ -32,35 +32,41 @@
 // except for one line of code.  Instead of duplicating the class, we should pull the CAVH
```

Comment:
I do understand why this class exists here but we should a) compare it to the most recent Weld 4.0 version and b) see if the Weld guys would be willing to make changes if we need them.
https://github.com/weld/core/blob/master/modules/jsf/src/main/java/org/jboss/weld/module/jsf/ConversationAwareViewHandler.java